holidays/countries/pakistan.py

```
@@ -25,14 +25,25 @@ class Pakistan(HolidayBase, InternationalHolidays, IslamicHolidays):
         * <https://en.wikipedia.org/wiki/Public_holidays_in_Pakistan>
         * <https://ur.wikipedia.org/wiki/تعطیلات_پاکستان>
     """
-
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)